components/AppBar/src/MDCAppBarViewController.m

```diff
@@ -172,6 +172,7 @@ - (void)setHeaderStackViewOffset:(CGFloat)headerStackViewOffset {
   _headerStackViewOffset = headerStackViewOffset;
   _verticalConstraint.constant = [self verticalContraintLength];
   _topSafeAreaConstraint.constant = [self topSafeAreaContraintLength];
+  self.shouldAdjustHeightBasedOnHeaderStackView = YES;
```

Comment:
This property is publicly exposed. Should we be changing it from it's client set value?